remote/advancedcontainers/docker-options.md

```diff
@@ -0,0 +1,110 @@
+---
+Order: 16
+Area: advancedcontainers
+TOCTitle: Docker Options
+PageTitle: Remote - Containers Docker options
+ContentId:
+MetaDescription: Remote - Containers Docker options
+DateApproved: 2/4/2022
+---
+
+# Alternate ways to install Docker
+
+You may harness Docker in the Remote - Containers extension in a few ways:
+* Docker installed locally
+* Docker installed on another machine or remote environment
+     * You'll just need Docker installed on the remote host, rather than needing to install Docker locally.
+* Other Docker compliant CLIs, installed locally or in a remote environment
+     * Remote - Containers interacts with CLIs; it makes no assumptions about how a container engine works and does not interact with container engines or daemon directly.
+     * Please note that other Docker compliant CLIs are not officially supported.
+
+Continue reading to learn alternate ways you may install and leverage Docker or a Docker compliant CLI.
+
+## Windows: Windows Subsystem for Linux (WSL)
+On Windows, you can use Docker installed in WSL through the [Remote - WSL extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-wsl). You can reference the [Docker documentation](https://docs.docker.com/engine/install/) for installing Docker on Linux, with specific information per distro starting [here](https://docs.docker.com/engine/install/centos/).
+
+One issue is that the dockerd daemon won't start automatically due to the lack of systemd or any other system daemon. You can use [Distrod](https://github.com/nullpo-head/wsl-distrod) to get WSL distros with systemd. You may install a new distro or update your current distros to run systemd.
+
+## MacOS: Colima
+
+[Colima](https://github.com/abiosoft/colima) provides container runtimes on macOS. It's recommended to use Colima v0.2.2 or later so that VS Code can properly see containers running through Colima.
+
+Colima automatically sets up a `colima` [context](https://docs.docker.com/engine/context/working-with-contexts/) in Docker and makes it the active context. You may also want to install the `docker` and `docker-compose` CLIs before running `colima start` for this setup to work properly.
+
+> Note: Colima uses Alpine Linux, which isn't supported by Remote - SSH.
+
+## Linux
+
+### Cloud-Init VM
+You may use the [Remote - SSH extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-ssh) with Remote - Containers. This enables you just to have Docker installed on your remote machine.
+
+You may use Remote - SSH to connect to a Linux VM with Docker installed. You may use a [Cloud-Init](https://cloud-init.io/) file (which is an industry standard) to install Docker on the VM automatically. As an example, you can create an Azure VM through the Azure CLI, and set it to use a `cloud-init.txt` during creation.
```

Comment:
Maybe remove first sentence because this is captured in the above paragraph?

Reply:
I agree this sounds repetitive. Removed the first sentence.